README.md

```diff
@@ -228,7 +228,7 @@ You can add these options to `$FZF_DEFAULT_OPTS` so that they're applied by
 default. For example,
 
 ```sh
-export FZF_DEFAULT_OPTS='--height 40% --reverse --border'
+export FZF_DEFAULT_OPTS='--height 40% --layout=reverse-list --border'
```

Comment:
I want to present `--layout=reverse` as an example instead of the new `reverse-list`.